pkg/cmd/server/api/validation/oauth.go

```diff
@@ -159,6 +162,22 @@ func ValidateLDAPIdentityProvider(provider *api.LDAPPasswordIdentityProvider) Va
 	return validationResults
 }
 
+func ValidateKeystoneIdentityProvider(provider *api.KeystonePasswordIdentityProvider, identityProvider api.IdentityProvider) ValidationResults {
+	validationResults := ValidationResults{}
+
+	if len(provider.RemoteConnectionInfo.URL) == 0 {
```

Comment:
should be able to just call ValidateRemoteConnectionInfo, which will check the url, ca, and client cert stuff